### PR TITLE
Feature/semantic UI elements

### PR DESCRIPTION
**Pull request by oleksii-sakun**

### CODE REVIEW

File: src/redux/reducers/inputReducer.js [hunk 1]
```diff
@@ -31,6 +37,41 @@ export default function inputReducer(state = initialState, action) {
...
+        ),
+        tasksInProgress: state.tasksInProgress.concat([action.payload]),
+      };
+
+    case DELETE_TASK_FROM_DONE:
+      return {
+        ...state,
+        tasksDone: state.tasksDone.filter(
+          (task) => task.id !== action.payload.id
+        ),
```
Comment (ArchieMollaev) [inline src/redux/reducers/inputReducer.js]: create function to reuse `state.tasksDone.filter(
          (task) => task.id !== action.payload.id
        )`
 kinda function filterList(list, taskId): filteredList

File: src/redux/reducers/inputReducer.js [hunk 2]
```diff
@@ -31,6 +37,41 @@ export default function inputReducer(state = initialState, action) {
...
+      };
+
+    case MOVE_TASK_TO_IN_PROGRESS_FROM_DONE:
+      return {
+        ...state,
+        tasksDone: state.tasksDone.filter(
+          (task) => task.id !== action.payload.id
+        ),
+        tasksInProgress: state.tasksInProgress.concat([action.payload]),
+      };
```
Comment (ArchieMollaev) [inline src/redux/reducers/inputReducer.js:56]: function addTaskToList(list, task): listWithNewTask

File: src/redux/reducers/inputReducer.js (hunk 1 above)
Comment (oleksii-sakun) [reply src/redux/reducers/inputReducer.js]: fixed

File: src/redux/reducers/inputReducer.js (hunk 2 above)
Comment (oleksii-sakun) [reply src/redux/reducers/inputReducer.js:56]: fixed

**Pull request merged**